our-app/src/components/authenticated/ImmunizationForm.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import { Form, Button, Card, Row, Col } from 'react-bootstrap';
+
+
+const ImmunizationForm = (props) => {
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const form = e.target;
+
+        const immunization = {
+            vaccinationId: form.vaccinationId.value,
+            patientId: form.patientId.value,
+            vLocation: form.vLocation.value,
+            vContraindications: form.vContraindications.value,
+            vContradictionsDate: form.vContradictionsDate.value,
+            vEligibility: form.Eligibility.value,
+            vExceptionDate: form.exceptionDate.value,
+            vExceptionReason: form.exceptionReason.value,
+            vOrgOrigination: form.vOrgOrigination.value,
+            vOrgSubmitter: form.vOrgSubmitter.value,
+            vAdminDate: form.vAdminDate.value,
+            vaccinationIdSubmitting: form.vaccinationIdSubmitting.value,
+            vEventType: form.vEventType.value,
+            vProviderPersonName: form.vProviderPersonName.value,
+            vProviderPersonSuffix: form.vProviderPersonSuffix.value,
+            vDoseVolume: form.vDoseVolume.value,
+            vDoseVolumeUnits: form.vDoseVolumeUnits.value,
+            vExpirationDate: form.vExpirationDate.value,
+            vFundingSource: form.vFundingSource.value,
+            vInfoStatementPubDate: form.vInfoStatementPubDate.value,
+            vInfoStatementDate: form.vInfoStatementDate.value,
+            vLotNum: form.vLotNum.value,
+            vMfgName: form.vMfgName.value,
+            vOrderingProvider: form.vOrderingProvider.value,
+            vProduct: form.vProduct.value,
+            vRouteOfAdmin: form.vRouteOfAdmin.value,
+            vSiteOfAdmin: form.vSiteOfAdmin.value
+
+        }
+        props.ImmunizationForm(immunization, props.history);
```

Comment:
What is this line? This is where we would pass the form to an action that would handle the submission but this is not set up yet, correct?

Reply:
Yup! We will dispatch this immunization object when we have it set up

Reply:
Should we remove line 40 since props.ImmunizationForm should be undefined right now?

Reply:
Yes I will just remove it for now